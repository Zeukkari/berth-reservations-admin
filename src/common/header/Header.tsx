import React from 'react';
import classNames from 'classnames';

import styles from './header.module.scss';

interface Props {
  brand?: React.ReactNode;
  navLinks?: React.ReactNode;
  className?: string;
}

const Header = ({ brand, className }: Props) => (
  <header className={classNames(styles.header, className)}>{brand}</header>
);

export default Header;
