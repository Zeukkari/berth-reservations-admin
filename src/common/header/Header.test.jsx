import { shallow } from 'enzyme';
import React from 'react';

import Header from './Header';
describe('common/Header', () => {
  it('renders a Header component', () => {

    const dummyProps = {
      brand: <h3>Asiakastiedot</h3>
    }

    const getWrapper = props => {
      return shallow(<Header {...dummyProps}></Header>);
    };
    const card = getWrapper();
    expect(card.html()).toMatchSnapshot();
  });
});
