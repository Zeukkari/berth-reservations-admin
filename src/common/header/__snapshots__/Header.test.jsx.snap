// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common/Header renders a Header component 1`] = `"<header class=\\"header\\"><h3>Asiakastiedot</h3></header>"`;
